pyNN/neuron/standardmodels/electrodes.py

```diff
@@ -210,8 +210,9 @@ def __init__(self, **parameters):
     def _generate(self):
         # Not efficient at all... Is there a way to have those vectors computed on the fly ?
         # Otherwise should have a buffer mechanism
-        self.times = numpy.arange(self.start, self.stop + simulator.state.dt, simulator.state.dt)
-        tmp = numpy.arange(0, self.stop - self.start, simulator.state.dt)
+        temp_num_t = int(round(((self.stop + simulator.state.dt) - self.start) / simulator.state.dt))
+        self.times = numpy.array([self.start + (i * simulator.state.dt) for i in range(temp_num_t)])
+        tmp = numpy.array([i * simulator.state.dt for i in range(temp_num_t)])
```

Comment:
as above, should use `numpy.arange`. In addition `self.times = tmp + self.start`, so should generate `tmp` first